dangerfile.js

```diff
@@ -24,13 +24,15 @@ const knownFrameworks = [
   'BackBone.js',
   'Box2D',
   'Bullet3',
+  'ClanLib',
   'CreateJS',
   'Cocos2d',
   'Construct',
   'Construct2',
   'Crystal Space',
   'Cube 2 Engine',
   'Daemon Engine',
+  'DarkBASIC',
```

Comment:
```suggestion
```
I think it's ok to just have darkbasic as a language, no need to also add it as framework